docs/blog/posts/announcing-keptn-v2-release-candidate.md

```diff
@@ -0,0 +1,148 @@
+---
+date: 2024-03-13
+authors: [agardnerit,bacherfl]
+description: >
+  Announcing the Official Release Candidate for Keptn v2
+categories:
+  - Installation
+  - Upgrade
+comments: true
+---
+
+# Announcing the Official Release Candidate for Keptn v2
+
+The Keptn project is proud to announce a release candidate for what will become Keptn v2.
+This release candidate provides the project with enough end-user validation
+and bug-finding so that we can release Keptn v2.
```

Comment:
```suggestion
We are sharing this release candidate
so that the community and end users can participate in the testing
to ensure that the actual release is as robust as possible.
```
I'm not in love with my rewrite but the existing sentence is just not true.  The release candidate does nothing to ensure the quality of the release unless people actually try it out.  Perhaps you can word this better.